### PR TITLE
Upgrade Camel Plugin eclipse/che#13921

### PR DESCRIPTION
**Pull request by apupier**
- provide 0.0.15 version
- publisher has moved from camel-tooling to redhat

PR for reference, cannot test locally for now

### CODE REVIEW

**Comment by l0rd** [comment]
@apupier what's the plan for this PR?

**Comment by apupier** [comment]
the plan is to test that the upgrade is working correctly as there is the specificity of modified publisher.  I plan to do it in September. (but if someone else is checking it before that's great too :-) )

File: v3/plugins/camel-tooling/vscode-apache-camel/0.0.14/meta.yaml [hunk 1]
```diff
@@ -10,6 +10,9 @@ icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
 repository: https://github.com/camel-tooling/camel-lsp-client-vscode
 category: Language
 firstPublicationDate: "2018-04-23"
+deprecate:
+  automigrate: true
+  migrateTo: redhat.vscode-apache-camel
```
[the remark this answers is not in the document]
Comment (apupier) [reply v3/plugins/camel-tooling/vscode-apache-camel/0.0.14/meta.yaml]: the version with redhat identifier is 0.0.16

[the remark this answers is not in the document]
Comment (apupier) [reply v3/plugins/camel-tooling/vscode-apache-camel/0.0.14/meta.yaml]: I updated with correct id to latest

File: v3/plugins/camel-tooling/vscode-apache-camel/latest/meta.yaml [hunk 2]
```diff
@@ -10,6 +10,9 @@ icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
 repository: https://github.com/camel-tooling/camel-lsp-client-vscode
 category: Language
 firstPublicationDate: '2018-04-23'
+deprecate:
+  automigrate: true
+  migrateTo: redhat.vscode-apache-camel
```
[the remark this answers is not in the document]
Comment (apupier) [reply v3/plugins/camel-tooling/vscode-apache-camel/latest/meta.yaml]: updated, thanks

**Comment by apupier** [comment]
I tested the latest plugins with the new publisher (redhat instead of camel-tooling) by modifying teh workspace devfile to

 ```
 - id: >-
      https://raw.githubusercontent.com/apupier/che-plugin-registry/13921-upgradeCamelPlugin/v3/plugins/redhat/vscode-apache-camel/latest/meta.yaml
    memoryLimit: 200Mi
    type: chePlugin
```

I'm unable to test the migrateTo part as I'm unable to provide my own registry.

@l0rd @rhopp @skabashnyuk @vparfonov can one of the code owner review the PR please so that I can potentially merge it?

**Pull request merged**